yt_idv/shaders/shaderlist.yaml

```diff
@@ -227,6 +234,13 @@ component_shaders:
       first_fragment: transfer_function
       second_vertex: passthrough
       second_fragment: passthrough
+    slice:
+      description: Slice
+      first_vertex: grid_position
+      first_geometry: grid_expand
+      first_fragment: slice_sample
+      second_vertex: passthrough
+      second_fragment: apply_colormap
```

Comment:
This is likely my fault from all the merging/rebasing.... but I **think** this should be up in the normal `block_rendering` and not the `octree_block_rendering`? Is that right?